src/README.md

```diff
@@ -0,0 +1,220 @@
+## Travel Tracker
+
+### Overview
+Travel Tracker is an application which manages and tracks different trips for 
```

Comment:
Say something like 
Travel tracker is an app that manages and tracks different trips for users and travel agents, by consuming and posting to an external API